data/events.xml

```diff
@@ -1,5 +1,141 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <events>
+     <event>
+        <lang>zh</lang>
+        <startDate>2021-11-19</startDate>
+        <endDate>2021-11-19</endDate>
+        <location>Taipei, Taiwan</location>
+        <speaker>Brandy Chang</speaker>
+        <title>SJCConf 2021</title>
+        <subject>Going Reactive with Kotlin and Quarkus</subject>
+        <url>https://jcconf.tw/2021/</url>
+        <description>
+            <![CDATA[
+<p>Introduce Quarkus for cloud native development.  Demo to build small and high concurrent restful service with Kotlin coroutines and Quarkus.<p>
+]]>
+        </description>
+      </event>
```

Comment:
@TinaNEs not sure if you saw it, but there is at least one more talk about KMM. Could you please ad it too?

Reply:
added 2 more talks with Kotlin